airlock_processor/local.settings.json-sample

```diff
@@ -0,0 +1,12 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "python",
+    "AIRLOCK_STATUS_CHANGED_QUEUE_NAME": "status_changed",
+    "SB_CONNECTION_STRING": "Endpoint=sb://XXXX.servicebus.windows.net/;SharedAccessKeyName=.....",
+    "AZURE_SUBSCRIPTION_ID": "",
+    "ENABLE_LOCAL_DEBUG": "true",
```

Comment:
Can this be using an env var we already have (like DEBUG or ENABLE_LOCAL_DEBUGGING)?